Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -197,7 +202,9 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
     uuid_t uuid;
     uuid_generate(uuid);
 
+#if !USE(EXTENSIONKIT)
     xpc_connection_set_oneshot_instance(m_xpcConnection.get(), uuid);
```

Comment:
Does this allow us to block anything in the sandbox?

Reply:
That is a good point! I do not think that is the case, since this will not affect e.g. which syscalls are being used, I believe. But that could very well be incorrect, and it would be good to check this closer.

Thanks for reviewing!